local-cli/eject/eject.js

```diff
@@ -64,6 +66,15 @@ function eject() {
     process.exit(1);
   }
 
+  const appIcon = appConfig.icon;
+  if(!appIcon) {
```

Comment:
The formatting is inconsistent, should be `if (`

The app icon should not be mandatory, we should have a fallback icon